packages/phlo-observatory/src/phlo_observatory/src/observatory/design/ledger.css

```
@@ -0,0 +1,2006 @@
+html[data-phlo-observatory-route='true'],
+.phlo-observatory {
+  --v2-font-xs: 12px;
+  --v2-font-sm: 13px;
+  --v2-font-base: 15px;
+  --v2-font-md: 18px;
+  --v2-font-lg: 22px;
+  --v2-font-xl: 30px;
+  --v2-weight-regular: 400;
+  --v2-weight-medium: 500;
+  --v2-weight-strong: 600;
+  --v2-sans: -apple-system, BlinkMacSystemFont, 'Segoe UI', sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)